ci-operator/config/cri-o/cri-o/cri-o-cri-o-main__ci.yaml

```diff
@@ -0,0 +1,58 @@
+base_images:
+  coreos-centos-stream8:
+    name: stream8
+    namespace: coreos
+    tag: "8"
+  dev-scripts:
+    name: test
+    namespace: ocp-kni
+    tag: dev-scripts
+  openshift_release_golang-1.18:
+    name: release
+    namespace: openshift
+    tag: golang-1.18
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.18
+images:
+- dockerfile_literal: |
+    FROM base
+    COPY . .
+  from: src
```

Comment:
I'm not sure if this stanza is actually building anything here.

Reply:
this is correct... it moves the code into an image.